### ISSUE TITLE
Behaviour On/Off

### ISSUE THREAD

**Issue by maverick74** [ericchase/maximize-all-windows#5]
it would be good to have an easy way to enable/disable de addon (without having to go to the addon preferences).

Here is a suggestion:
Why not on Middle click on the button it would maximize all windows
and on left click it would enable or disable the function

the point is that i don't want windows maximized all the time, just when i'm doing a specific kind of work.

So a easy way to enable/disable would be good.
ATM i have to go to about:addons and enable/disable the addon

thank you for your time and work, by the way :)

**Comment by ericchase** [comment]
Yes, this is a good idea. I'm sure it's an easy add. I'll see if I can do it when I find time

**Comment by maverick74** [comment]
If you're able to find some, tell me something will you?!
I'm searching for it as well ahahahah

**Comment by ericchase** [comment]
Ok, I moved the options into the right click menu the addon action icon as checkboxes. No more options page. I also added an Enable/Disable checkbox.

![image](https://user-images.githubusercontent.com/30473157/169678730-6c34f435-3c32-4a53-abb7-d57c380c37ac.png)

Hopefully this will be good enough.

**Comment by ericchase** [comment]
Due to Google enforcing their **manifest v3**, I am in the process of rewriting the entire addon. I think I will leave the original addon how it is, and add the new version as a new addon to both the Chrome and Firefox stores.

Someone else requested a new feature that I want to add as well.
And lastly, I want to try dealing with the ugly icon.

I hope to finish everything by next week, but we will see.

**Comment by maverick74** [comment]
> add the new version as a new addon to both the Chrome and Firefox stores.
> 
> Someone else requested a new feature that I want to add as well. And lastly, I want to try dealing with the ugly icon.
> 
> I hope to finish everything by next week, but we will see.

Nice news. Please, when you have it share the link to the new add-on so that I can switch to it :)

thanks

**Comment by maverick74** [comment]
@ericchase Any news?

**Comment by ericchase** [comment]
Sorry man, things did not go as projected
Honestly, I've been on hiatus from coding for a while
I might just have to split the new project up into parts and just push the minimal working code for now, and get back to the rest at a later time

**Comment by ericchase** [comment]
I'll publish the new extension in beta mode if that's still a thing. Hopefully it will just work like before for now.

**Comment by ericchase** [comment]
Ok, I did my best. Manifest v3 changed a lot of stuff around, so I did what I could to keep things similar to before.

If it gets approved, I'll let you know.

**Comment by ericchase** [comment]
https://chrome.google.com/webstore/detail/maximize-windows/oealklhjfdmeafmhkmgjplhlldkiblge

Welp, here it is. Please let me know if you find any bugs

**Comment by maverick74** [comment]
@ericchase Thank you a lot for your time!!!

I understand how often "real life" gets in the way :).

I won't be able to test it, however. I'm sorry.

I'm on Firefox and won't get near chrome even if google pays me!

From what I've learned Manifest v3 is major pain for devs (specially adblock devs)

**Comment by ericchase** [comment]
> From what I've learned Manifest v3 is major pain for devs (specially adblock devs)...and still changing...

Yes, it seems so. I'm not sure how Firefox plans to deal with mv3. In any case, I've tried writing the code in a flexible manner. With some future adjustments, I should be able to generate versions for both Chrome and Firefox at the same time, which is the goal, since I use Firefox myself. I originally thought that mv3 would be available by now on Firefox, but they've pushed it back to next year seemingly. I will try to work on cross platform adjustments when I find the energy to

**Comment by maverick74** [comment]
Firefox delayed it, yes.
But I was under the impression you could already develop add-ons based on game publish them on AMO.

But they also choose not to implement all mv3 because it cripples devs too much...
